README.md

```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
 ### Create reCAPTCHA account
 [Create ReCAPTCHA account](http://www.google.com/recaptcha/admin)
 
-[Create Github OAuth App](https://github.com/settings/developers)
+### Create Github OAuth Application
+
+1. Register in [Github OAuth](https://github.com/settings/developers) with this following examples:
+
+| Field                      | Value                                                    |
+| -------------------------- | -------------------------------------------------------- |
+| Application Name           | `short`                                                  |
+| Homepage URL               | `https://s.time4hacks.com`                               |
+| Application description    | `URL shortening service written in Go and React`         |
+| Authorization callback URL | `https://s.time4hacks.com/oauth/github/sign-in/callback` |
```

Comment:
I wonder whether we want to change `https://s.time4hacks.com` to `http://localhost` in order to easy local testing.

Reply:
Yeah, agreed